viral/offline_sequence_detection.py

```
@@ -0,0 +1,707 @@
+    of the bins observed within a given event."
+    "'timeBinPermutation': permutes (resamples without replacement)"
+    # TODO Essentially, this is a Python implementation of https://github.com/losonczylab/Grosmark_NatNeuro_2021/blob/main/shufflePopulationEvents.m time bin permutation
+    """
+    # TODO: check actual shape
+    # TODO: make this an empirical P value function?
+    return shuffle_rows(posterior_probability_matrix)
+
+
+def plot_pse_event(
+    posterior_probability_matrix: np.ndarray,
+    pr_max: np.ndarray,
```

Ruff (0.14.3)

Unused function argument: `pr_max`

(ARG001)